### ISSUE TITLE
Results on OpenSSL_min50

### ISSUE THREAD

**Issue by hyp1231** [cszhangzhen/H2MN#3]
Hi,

Thanks for the interesting work and great open-source!

However, when I tried to reproduce the results on openssl_min50, I run into some difficulties.

I used the released dataset (openssl_min50), run with several scripts, and always got results far away from those reported in the original paper (AUC 96.80±0.95). I wonder if I had missed some important details.

-----

```bash
python main_classification.py --dataset openssl_min50 --batch-size 16 --learning-rate 0.001 --ratio1 0.7 --dropout 0.5
```

I got AUC 90.61 at the best epoch 228.

-----

```bash
python main_classification.py --dataset openssl_min50 --batch-size 16
```

I got AUC 85.88 at the best epoch 159.

-----

```bash
python main_classification.py --dataset openssl_min50 --batch-size 16 --learning-rate 0.001 --ratio1 0.7 --ratio2 0.7 --ratio3 0.7 --dropout 0.5
```

I got AUC 78.78 at the best epoch 210.

----

The environment is similar to the requirements:

```
python==3.7.7
pytorch==1.7.1
torch_geometric==1.6.3
torch_scatter==2.0.7
torch_sparse==0.6.9
torch_cluster==1.5.9
sklearn==0.24.1
numpy==1.19.2
scipy==1.6.0
```

Thanks for your time and looking forward to the suggestions!

**Comment by cszhangzhen** [comment]
Hi,

Try the following parameters:
lr = 0.0001, dropout=0, ratio1 = ratio2 = 1.0, ratio3 = 0.7

**Comment by hyp1231** [comment]
Thanks for the quick response and suggestion. It gets

```
Test set results, loss = 0.376884, AUC = 0.911020
```

and still not close to 96.80.

**Comment by cszhangzhen** [comment]
It achieves 0.9632 on my server.
<img width="955" alt="image" src="https://user-images.githubusercontent.com/39481550/159112692-80385cb0-71a0-403e-823c-b1cc1e59e92d.png">

**Comment by hyp1231** [comment]
Thanks! I'll change the server (and environments) and try again.

**Comment by hyp1231** [comment]
I created a new environment that strictly follows the package requirements (except `numpy`, which requires at least `1.18.0` to run successfully). Then I got
```
Epoch: 00472, loss_train: 0.182599, loss_val: 0.203843, AUC: 0.984389, time: 3519.568021s
Epoch: 00473, loss_train: 0.204938, loss_val: 0.333375, AUC: 0.944904, time: 3527.266397s
Epoch: 00474, loss_train: 0.246946, loss_val: 0.245485, AUC: 0.961433, time: 3534.349454s
Optimization Finished! Total time elapsed: 3534.379762

Model evaluation.
Test set results, loss = 0.356469, AUC = 0.940408
```

which is close enough for me. Maybe the versions of packages indeed affect the performance of H2MN.  :)

Overall, thanks for the suggestions!

**Comment by cszhangzhen** [comment]
Hi,

Feel free to contact me if you have any other problems. Do not know why different versions cause the variances.
